third_party/amd/lib/TritonAMDGPUTransforms/ConvertToBufferOps.cpp

```diff
@@ -491,10 +746,12 @@ class TritonAMDGPUConvertToBufferOpsPass
       if (op->getOperand(0).getDefiningOp<arith::CmpIOp>())
         assumptions.insert(op->getOperand(0));
     });
-    LDBG("Number of assumptions found: " << assumptions.size());
```

Comment:
These debugging prints are still useful? Why deleting them?

Reply:
to be honest i thought they were in fact not useful because they're just printing things that are visible in the IR already. but sure i can restore them.